inc/issue.class.php

```diff
@@ -521,7 +522,16 @@ public static function giveItem($itemtype, $option_id, $data, $num) {
       switch ("$table.$field") {
          case "glpi_plugin_formcreator_issues.name":
             $name = $data[$num][0]['name'];
-            return "<a href='".FORMCREATOR_ROOTDOC."/front/issue.form.php?id=".$id."&sub_itemtype=".$data['raw']['sub_itemtype']."'>$name</a>";
+            $ticket->getFromDB($data[$num][0]['id']);
+            $content = $ticket->getField('content');
+            $desc = $data[$num][0]['desc'];
+            return "<a id='Ticket".$data[$num][0]['id']."' href='".FORMCREATOR_ROOTDOC."/front/issue.form.php?id=".$id."&sub_itemtype=".$data['raw']['sub_itemtype']."'>$name</a><div style='display:none' id='TicketContent".$data[$num][0]['id']."'>".nl2br($content)."</div><script type='text/javascript'>
+            //<![CDATA[
```

Comment:
Have a look into Html::showToolTip() . It is better to use this method instead of duplicating its feature.